unlock-app/src/components/interface/locks/metadata/utils.ts

```diff
@@ -1,5 +1,7 @@
 export interface Certification {
   certification_issuer?: string
+  Expiration?: number
+  Minted?: number
```

Comment:
```suggestion
  Expiration?: number
  Minted?: number
```

nit: Make this lowercase to keep consistency?

Reply:
Good point